src/golang/lib/engine/notification.go

```diff
@@ -12,8 +12,11 @@ import (
 )
 
 type notificationContentStruct struct {
-	level      mdl_shared.NotificationLevel
-	contextMsg string
+	level mdl_shared.NotificationLevel
+	// Additional system error that the user should be notified.
+	// Execution related user errors should already captured by
+	// dag.WorkflowDag object.
```

Comment:
What is this referring to? I don't see anything error-related on the WorkflowDag interface.